libs/deps/plans-modal.js

```diff
@@ -4459,6 +4459,7 @@ var setCustomTemplateLiteralTag2 = (e16) => {
 
 // ../node_modules/@spectrum-web-components/icons-workflow/src/icons/Alert.js
 var AlertIcon = ({ width: a10 = 24, height: t13 = 24, hidden: e16 = false, title: r9 = "Alert" } = {}) => tag2`<svg
+    aria-hidden="true"
```

Comment:
Already defined dynamically on L4467, not needed here as well.

Reply:
Fixed 👍 Thank you 🙏🏻